optuna_dashboard/ts/components/GraphIntermediateValues.tsx

```diff
@@ -51,19 +51,23 @@ const plotIntermediateValue = (trials: Trial[], mode: string) => {
     return
   }
 
-  const filteredTrials = trials.filter(
+  const FilteredTrials = trials.filter(
```

Comment:
In Optuna Dashboard, we name local variables in camelCase.

```suggestion
  const filteredTrials = trials.filter(
```